app/models/concerns/publishable.rb

```diff
@@ -15,7 +15,7 @@ def workflow_status
   end
 
   def publish!(user_id = nil)
-    user = User.find(user_id) if user_id
+    user = User.where(id: user_id).first if user_id
```

Comment:
Why are you doing this?  We want to raise an error if they supply an id of a nonexistent user by raising an error.